docs/changelog-fragments/25.bugfix.rst

```diff
@@ -0,0 +1 @@
+Changed pylibssh dir to pylibsshext to avoid ns collision -- by :user:`ganeshrn`
```

Comment:
It's not a bugfix. We just renamed the importable. There was no collision. We wanted the name to be more clear AFAIR. If anything, I'd add it as a feature.